isis/CMakeLists.txt

```diff
@@ -452,11 +452,22 @@ execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/inc)
 # Create the appdata directory and translations directory
 execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/translations)
 execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/templates/apollo)
+execute_process(COMMAND mkdir -p ${CMAKE_BINARY_DIR}/appdata/templates/autoreg) # *.def
```

Comment:
This is the non-ideal way of doing this in cmake that is functional, but not pretty. Punted a better solution to later.

Reply:
We'll add it to the todo list to make sure it doesn't fall through the cracks

Reply:
That sounds very good, thank you!